Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
old sidebar brings the whole blog list and everything back, old stats brings follower and post count back afaik. So yeah, they do like pretty different